test/tests/functional/pbs_qstat.py

```diff
@@ -57,6 +58,61 @@ def parse_json(self, dictitems, qstat_attr):
                         self.parse_json(val, qstat_attr)
         return qstat_attr
 
+    def validate_el(self, attr_dict, user):
+        """
+        Check if the value of server attributes reported by
+        qstat -Bf matches the expected values.
+        """
+        qstat_o = {}
+        qstat_o['server_state'] = 'Active'
+        qstat_o['server_host'] = self.server.hostname
+        qstat_o['scheduling'] = 'True'
+        qstat_o['total_jobs'] = '0'
+        qstat_o['state_count'] = ['Transit', 'Queued',
+                                  'Held', 'Waiting', 'Running',
+                                  'Exiting', 'Begun']
+        qstat_o['default_queue'] = 'workq'
+        qstat_o['log_events'] = '511'
+        qstat_o['mail_from'] = 'adm'
+        qstat_o['query_other_jobs'] = 'True'
+        qstat_o['resources_default.ncpus'] = '1'
+        qstat_o['default_chunk.ncpus'] = '1'
+        qstat_o['scheduler_iteration'] = '600'
+        qstat_o['resv_enable'] = 'True'
+        qstat_o['node_fail_requeue'] = '310'
+        qstat_o['max_array_size'] = '10000'
+        qstat_o['pbs_license_linger_time'] = '3600'
+        qstat_o['pbs_version'] = self.server.pbs_version
+        qstat_o['eligible_time_enable'] = 'False'
+        qstat_o['max_concurrent_provision'] = '5'
+
+        if user == ROOT_USER:
+            qstat_o['power_provisioning'] = 'False'
+
+        platform = self.du.get_platform()
+        if platform == 'cray' or platform == 'craysim':
+            qstat_o['restrict_res_to_release_on_suspend'] = 'ncpus'
+
+        for each in attr_dict:
+            if each in qstat_o:
```

Comment:
This won't determine if there is an attribute missing from attr_dict that is in qstat_o.